packages/drivers/odsp-driver/src/odspPublicUtils.ts

```diff
@@ -26,3 +28,21 @@ export interface ISnapshotContents {
 	 */
 	latestSequenceNumber: number | undefined;
 }
+
+/**
+ * Picks out the pieces necessary for sending a request to the broadcast-signal endpoint on PUSH.
+ * Returns ISocketStorageDiscoveryParts.
+ * @param socketStorageDiscovery - The (raw) ISocketStorageDiscovery to parse
+ */
+export function getSocketStorageDiscoveryParts(
+	socketStorageDiscovery: ISocketStorageDiscovery | undefined,
+): ISocketStorageDiscoveryParts | undefined {
+	if (socketStorageDiscovery !== undefined) {
+		return {
+			runtimeTenantId: socketStorageDiscovery?.runtimeTenantId,
+			tenantId: socketStorageDiscovery.tenantId,
+			deltaStreamSocketUrl: socketStorageDiscovery.deltaStreamSocketUrl,
+		};
+	}
+	return undefined;
```

Comment:
Is it important to create a new object here?  Or could we just return the same object?  Or just cast the object in the calling location?